CoreFoundation/Stream.subproj/CFStream.h

```diff
@@ -78,6 +78,8 @@ const CFStringRef kCFStreamPropertyDataWritten;
 CF_EXPORT
 CFReadStreamRef CFReadStreamCreateWithBytesNoCopy(CFAllocatorRef alloc, const UInt8 *bytes, CFIndex length, CFAllocatorRef bytesDeallocator);
 
+CF_EXPORT CFReadStreamRef CFReadStreamCreateWithData(CFAllocatorRef alloc, CFDataRef data);
```

Comment:
This probably should be a private method, putting it here makes it public API which I am not certain we really want to do just yet. Can you move this declaration to ForFoundationOnly.h or ForSwiftFoundationOnly.h?